AUTHORS.md

```diff
@@ -0,0 +1,28 @@
+# General credits
+
+NMRLipids project is run by [NMRLipids research collaboration](https://nmrlipids.blogspot.com/) since 2013.
+
+# Contributors
+
+NMRLipids Databank separates codespace from [Data](https://github.com/NMRLipids/BilayerData) since June 2025 (v.1.1.0). Here we list only the contributors of the code part. The contributors of the bilayer data for the Databank project are listed in [BilayerData/AUTHORS.md](https://github.com/NMRLipids/BilayerData/blob/main/AUTHORS.md).
```

Comment:
I would write:

Since June 2025 (v.1.1.0), NMRlipids Databank separates codespace from [Data](https://github.com/NMRLipids/BilayerData). Below we list contributors of the code part only; contributors of the bilayer-data part are listed at [BilayerData/AUTHORS.md](https://github.com/NMRLipids/BilayerData/blob/main/AUTHORS.md).